README.md

```diff
@@ -89,7 +89,9 @@ the repository.
 ## 🔭 Future Features <a name="future-features"></a>
 
 
-- I will set up the mobile version.
+- Modal/Pop up windows for the cards details.
+- Javascript to make page interactive.
+- Validate contact form and preserve data in the browser.
```

Comment:
Well done on using the README template 👍🏽  but make sure that all required sections of the README are present. Remember only the `FAQ` is optional 😉 

Also, make sure to add @Ruthmy as an `Author` in the corresponding README section 📄